### PR TITLE
5442 solr731

### PR DESCRIPTION
**Pull request by pameyer**
connects to #5442

## New Contributors

Welcome! New contributors should at least glance at [CONTRIBUTING.md](/CONTRIBUTING.md), especially the section on pull requests where we encourage you to reach out to other developers before you start coding. Also, please note that we measure code coverage and prefer you write unit tests. Pull requests can still be reviewed without tests or completion of the checklist outlined below. Thanks!

## Related Issues

- connects to #5422 solr minor version increment 

## Pull Request Checklist

- [ ] Unit [tests][] completed
- [ ] Integration [tests][]: None
- [ ] Deployment requirements, [SQL updates][], [Solr updates][], etc.: None
- [ ] [Documentation][docs] completed
- [ ] Merged latest from "develop" [branch][] and resolved conflicts

[tests]: http://guides.dataverse.org/en/latest/developers/testing.html
[SQL updates]: http://guides.dataverse.org/en/latest/developers/sql-upgrade-scripts.html
[Solr updates]: https://github.com/IQSS/dataverse/blob/develop/conf/solr/7.3.0/schema.xml
[docs]: http://guides.dataverse.org/en/latest/developers/documentation.html
[branch]: http://guides.dataverse.org/en/latest/developers/branching-strategy.html

### CODE REVIEW

File: conf/docker-aio/entrypoint.bash [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
-
+export LANG=en_US.UTF-8
```
Comment (pdurbin) [inline conf/docker-aio/entrypoint.bash]: What's this LANG environment variable for? Should we add a comment?

File: conf/solr/7.3.1/readme.me [hunk 2]
```diff
@@ -0,0 +1 @@
+Please see the dev guide for what to do with Solr config files.
```
Comment (pdurbin) [inline conf/solr/7.3.1/readme.me]: This should probably end in ".md" instead of ".me".

File: conf/solr/7.3.1/schema.xml [hunk 3]
```diff
@@ -0,0 +1,1639 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (pdurbin) [inline conf/solr/7.3.1/schema.xml]: Is this identical to the 7.3.0 schema?

File: conf/solr/7.3.1/solrconfig.xml [hunk 4]
```diff
@@ -0,0 +1,1410 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (pdurbin) [inline conf/solr/7.3.1/solrconfig.xml]: Is this identical to the 7.3.0 solrconfig?

File: scripts/vagrant/setup.sh [hunk 5]
```diff
@@ -40,7 +40,7 @@ echo "Ensuring Unix user '$SOLR_USER' exists"
 useradd $SOLR_USER || :
 DOWNLOAD_DIR='/dataverse/downloads'
 GLASSFISH_ZIP="$DOWNLOAD_DIR/glassfish-4.1.zip"
-SOLR_TGZ="$DOWNLOAD_DIR/solr-7.3.0.tgz"
+SOLR_TGZ="$DOWNLOAD_DIR/solr-7.3.1.tgz"
```
Comment (pdurbin) [inline scripts/vagrant/setup.sh]: For this to work we need to edit downloads/download.sh

File: conf/docker-aio/entrypoint.bash (hunk 1 above)
Comment (pameyer) [reply conf/docker-aio/entrypoint.bash]: Being defensive for cases where the host system is using a non-US, non-UTF-8 locale.

File: conf/solr/7.3.1/readme.me (hunk 2 above)
Comment (pameyer) [reply conf/solr/7.3.1/readme.me]: I agree - but this was how I found it in the 7.3.0 configs.

File: conf/solr/7.3.1/schema.xml (hunk 3 above)
Comment (pameyer) [reply conf/solr/7.3.1/schema.xml]: Good catch - they are identical, and the 7.3.0 directory should be deleted in this PR.

File: scripts/vagrant/setup.sh (hunk 5 above)
Comment (pameyer) [reply scripts/vagrant/setup.sh]: Another good catch.

File: conf/solr/7.3.1/readme.me (hunk 2 above)
Comment (pdurbin) [reply conf/solr/7.3.1/readme.me]: Weird. Can you please use "md" for 7.3.1?

**Pull request merged**